index.html

```diff
@@ -79,6 +79,212 @@ <h1 class="h1">I'm David. Glad to see you!</h1>
           </ul>
         </div>
       </section>
+
+      <section class="projects">
+        <div class="project-item">
+          <img
+            src="images/project1_snapshot.png"
+            alt="project1"
+            width="295"
+            height="220"
+          />
+          <h3 class="h1 bold">Tonic</h3>
+          <div class="captions">
+            <p class="bold">CANOPY</p>
+            <span></span>
+            <p class="medium light-grey">Back End Dev</p>
+            <span></span>
+            <p class="medium light-grey">2015</p>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="project-language">
+            <li class="medium dark-purple">HTML</li>
+            <li class="medium dark-purple">CSS</li>
+            <li class="medium dark-purple">JS</li>
+          </ul>
+          <button class="button-outlined medium">See project</button>
+        </div>
+        <div class="project-item">
+          <img
+            src="images/project2_snapshot.png"
+            alt="project1"
+            width="295"
+            height="220"
+          />
+          <h3 class="h1 bold">Multi-Post Stories</h3>
+          <div class="captions">
+            <p class="bold">CANOPY</p>
+            <span></span>
+            <p class="medium light-grey">Back End Dev</p>
+            <span></span>
+            <p class="medium light-grey">2015</p>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="project-language">
+            <li class="medium dark-purple">HTML</li>
+            <li class="medium dark-purple">CSS</li>
+            <li class="medium dark-purple">JS</li>
+          </ul>
+          <button class="button-outlined medium">See project</button>
+        </div>
+        <div class="project-item">
+          <img
+            src="images/project3_snapshot.png"
+            alt="project1"
+            width="295"
+            height="220"
+          />
+          <h3 class="h1 bold">Tonic</h3>
+          <div class="captions">
+            <p class="bold">CANOPY</p>
+            <span></span>
+            <p class="medium light-grey">Back End Dev</p>
+            <span></span>
+            <p class="medium light-grey">2015</p>

```

Comment:
Kindly make sure you use semantic tags in your markup, as you may notice giving these elements paragraph as the container doesn't have any semantic meaning right?. Think of a paragraph tag as the real paragraph in a given literature, text, or essay,...
`Hint: You may consider using a unorder list <ul><li></li><ul> to grab them since it's a list of elements.`